tests/commands/entry_break.py

```diff
@@ -0,0 +1,15 @@
+"""
+`entry-break` command test module
+"""
+
+
+from tests.utils import GefUnitTestGeneric, gdb_run_cmd
+
+
+class EntryBreakCommand(GefUnitTestGeneric):
+    """`entry-break` command test module"""
+
+
+    def test_cmd_entry_break(self):
+        res = gdb_run_cmd("entry-break")
+        self.assertNoException(res)
```

Comment:
Add test that if we run it 2x it says, "gdb is already running", although maybe that string should change.,.. should it say, "target is already running"?